vestido/vestido-storefront/modules/Wishlist/size-selector.tsx

```diff
@@ -135,7 +137,11 @@ export const SizeSelectorDialog: React.FC<SizeSelectorDialogProps> = ({
   return (
     <>
       <AlertDialog>
-        <AlertDialogTrigger asChild>{children}</AlertDialogTrigger>
+        {disabledTrigger ? (
+          children
+        ) : (
+          <AlertDialogTrigger asChild>{children}</AlertDialogTrigger>
```

Comment:
Why can't we simply pass `disabled` to AlertDialogTrigger? 
Isn't passing disabled on the button enough?